src/peft/utils/other.py

```diff
@@ -30,6 +30,8 @@
 def infer_device():
     if torch.cuda.is_available():
         torch_device = "cuda"
+    elif hasattr(torch.backends, "mps") and torch.backends.mps.is_available():
+        torch_device = torch.device("mps")
```

Comment:
What do you think of warning users here that the MPS support is experimental and might lead to unexpected behaviour with safetensors and/or bf16?